lesson4/hometask_frameq.rb

```diff
@@ -0,0 +1,29 @@
+class LazyHash < Hash
+  def method_missing(name)
+    key = name.to_s
+    super.method_missing(key) unless self.key?(key)
```

Comment:
The way how you calling `super` is incorrect. `#super` (_a method_) is not a `superclass` (_an object_).
You need to just call `super`. That's it.